src/main/java/com/example/springfirsthw/dto/CreateBookRequestDto.java

```diff
@@ -2,19 +2,20 @@
 
 import jakarta.validation.constraints.NotNull;
 import jakarta.validation.constraints.Positive;
+import jakarta.validation.constraints.Size;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class CreateBookRequestDto {
-    @NotNull(message = "can't be null")
+    @Size(min = 1, max = 50)
     private String title;
     @NotNull(message = "can't be null")
```

Comment:
It would be better to use `@NotBlank`. [Check](https://www.baeldung.com/java-bean-validation-not-null-empty-blank#the-@notblank-constraint) what is does.